webapp/src/js/services/format-data-record.js

```diff
@@ -180,13 +180,36 @@ angular.module('inboxServices').factory('FormatDataRecord',
               });
     };
 
-    var formatDate = function(settings, date) {
+    var capitalizeWord = function(str) {
+      return str.charAt(0).toUpperCase() + str.slice(1);
+    };
+
+    var fromNowOrToday = function(momento, today) {
+      var startOfDay = moment().startOf('day');
+      var duration = moment.duration(startOfDay.diff(momento));
+      if(duration.asHours < 24) {
+        return capitalizeWord(today);
+      }
+      return null;
+    };
+
+    var formatDate = function(settings, date, locale, field) {
       if (!date) {
           return;
       }
       var m = moment(date);
+
+      var fromNow = m.fromNow();
+      if(field === 'child_birth_date') {
```

Comment:
It'd be worth checking if there are any other field names we should check for too.

@abbyad Can you think of any other fields names for which a day precision date is desirable?